ocs_ci/deployment/deployment.py

```diff
@@ -898,6 +912,28 @@ def deploy_ocs_via_operator(self, image=None):
                 f"{constants.NODE_SELECTOR_ANNOTATION}"
             )
 
+    def deploy_odfaddon(self):
+        """
+        This method deploy ODF addon.
+
+        """
+        logger.info("Deploying odf with ocs addon.")
+        clustername = config.ENV_DATA.get("cluster_name")
+        cmd = f"ibmcloud ks cluster addon enable openshift-data-foundation --cluster {clustername} -f"
+        out = run_ibmcloud_cmd(cmd)
+        time.sleep(1200)
```

Comment:
Why we need to wait 20 mins here?

The rest of the verification is happening in deploy_ocs method here:
https://github.com/red-hat-storage/ocs-ci/blob/4e0350f2df0af83f84f7d50eb4a9cfa3febb0f11/ocs_ci/deployment/deployment.py#L1064

Where we are waiting for mon pods to appear, and so on and so fort.

So I think that once you run the command you should maximum wait for 2 mins and then return from this function and let the rest of the functionality in deploy_ocs to check if all is done. So the following code can be deleted I think.